Dockerfile.rhel7

```diff
@@ -7,10 +9,17 @@ COPY . .
 # just use that.  For now we work around this by copying a tarball.
 RUN make install DESTDIR=./instroot && tar -C instroot -cf instroot.tar .
 
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-4.13 AS rhel9-builder
```

Comment:
ah, this somehow got generated as 4.13...

Reply:
oh I see, the tests didn't re-trigger

Reply:
I forgot to run `make Dockerfile.rhel7`, done